Update docs for monorepo (#2111)
vercel/docs: pages/docs/git-integrations/index.mdx

In the future, this workflow piece will be improved drastically. In the meantime, we strongly suggest using one of our [Git Integrations](/docs/git-integrations) instead of Vercel CLI.

> ### FAQ

Reviewer: ```suggestion
### FAQ about Monorepos
```

> ### FAQ about Monorepos

Reviewer 2: ```suggestion
### Monorepo FAQ
```

<br />

<CollapseGroup>
  <Collapse small title="Are Yarn or Lerna Workspaces supported?">
    <div style={{ height: 10 }} />
    <p>Yes. They will work automatically.</p>
  </Collapse>
  <Collapse
    small
    title="How can I make my projects available on different paths under the same domain?"
  >
    <div style={{ height: 10 }} />
    <p>

>       After having set up your monorepo as described above, each of the

Reviewer 2: ```suggestion
      After setting up your monorepo as described above, each of the
```

>       directories will be a separate Vercel project, and therefore be available

Reviewer 2: ```suggestion
      directories will be a separate Vercel [Project](/docs/platform/projects), and therefore be available
```

>       directories will be a separate Vercel [Project](/docs/platform/projects), and therefore be available

Reviewer 3: ```suggestion
      directories will be a separate Vercel  <Link color href="/docs/platform/projects">Project</Link>, and therefore be available
```

In the future, this workflow piece will be improved drastically. In the meantime, we strongly suggest using one of our [Git Integrations](/docs/git-integrations) instead of Vercel CLI.

### Monorepo FAQ

<br />

<CollapseGroup>
  <Collapse size="small" title="How can I speed up builds?">
    <div style={{ height: 10 }} />
    <p>
      Whether or not your deployments are queued depends on the amount of
      Concurrent Builds you have available. Personal accounts are limited to 1
      Concurrent Build, while teams can customize the amount on the "Billing"

>       page in the team settings.

Reviewer 4: Probably link to `/knowledge/why-are-my-vercel-builds-queued` here

    </p>
  </Collapse>
  <Collapse
    size="small"
    title="Can I share source files between projects? Are Yarn & Lerna Workspaces supported?"
  >
    <div style={{ height: 10 }} />
    <p>

>       Vercel projects created after August 26th 2020 20:35:59 UTC are able to

Reviewer 4: I don't think this date makes sense anymore. Instead, it should explain the new checkbox

      access source files outside their Root Directory. This also means Yarn &
      Lerna Workspaces are supported by default.
    </p>
    <p>
      To take advantage of this behavior on existing projects, turn on the
      "Include source files outside of the Root Directory..." option on the Root
      Directory section in the project settings.
    </p>
  </Collapse>
  <Collapse size="small" title="How are projects built after I push?">
    <div style={{ height: 10 }} />
    <p>
      Pushing a commit to a Git repository that is connected with multiple
      Vercel projects will result in multiple deployments being created and

>       built in parallel for each.

Reviewer 4: This should link to docs explaining the limits on Concurrent Builds and hwo to increase Concurrent Builds. Probably `/knowledge/why-are-my-vercel-builds-queued`

    </p>
  </Collapse>
  <Collapse
    size="small"
    title="How can I make my projects available on different paths under the same domain?"
  >
    <div style={{ height: 10 }} />
    <p>
      After having set up your monorepo as described above, each of the
      directories will be a separate Vercel project, and therefore be available
      on a separate domain.
    </p>
    <p>

>       If you'd like to make multiple projects available as different paths under
>       the same domain, you can set
>       <Link href="/docs/configuration#project/rewrites" color>
>         rewrites
>       </Link>
>       for those paths on the project that contains the main domain.

Reviewer 4: ```suggestion
      If you'd like to host multiple Projects under a single domain, you can create a new Project with that domain that acts as a proxy to the other upstream Projects. Create a `vercel.json` with the
      {' '}<Link href="/docs/configuration#project/rewrites" color>rewrites</Link>{' '}
      property to map each path each Project's upstream domain.
```

    </p>
  </Collapse>
  <Collapse
    size="small"
    title="Can I prevent certain projects from being deployed when pushing?"
  >
    <div style={{ height: 10 }} />
    <p>
      At the moment, every commit will issue a deployment for all projects that
      are connected with your Git repository at that time.
    </p>
    <p>
      In the future, you will be able to select which paths should be considered

>       by the Git Integration when deciding whether to deploy a project.

Reviewer 4: ```suggestion
      by the <Link href="/docs/git-integrations">Git Integration</Link> when deciding whether to deploy a project.
```
